db/seeds.rb

```diff
@@ -13,26 +13,30 @@ def self.start
   end
 
   def generate_categories
-    Category.create!(name: "Produce")
-    puts "category Produce created!"
-    Category.create!(name: "Drinks")
-    puts "category Drinks created!"
-    Category.create!(name: "Plants")
-    puts "category Plants created!"
-    Category.create!(name: "Jewlery")
-    puts "category Jewlery created!"
-    Category.create!(name: "Crafts")
-    puts "category Crafts created!"
-    Category.create!(name: "Dairy")
-    puts "category Dairy created!"
-    Category.create!(name: "Baked Goods")
-    puts "category Baked Goods created!"
-    Category.create!(name: "Meats")
-    puts "category Meats created!"
-    Category.create!(name: "Herbs")
-    puts "category Herbs created!"
-    Category.create!(name: "Caffeine")
-    puts "category Caffeine created!"
+    Category.create!(name: "Fruits")
+    puts "category Fruits created!"
+    Category.create!(name: "Vegetables")
+    puts "category Vegetables created!"
+    Category.create!(name: "Home Goods")
+    puts "category Home Goods created!"
+    # Category.create!(name: "Drinks")
+    # puts "category Drinks created!"
+    # Category.create!(name: "Plants")
+    # puts "category Plants created!"
+    # Category.create!(name: "Jewelry")
+    # puts "category Jewelry created!"
```

Comment:
these were spelled incorrectly, so I edited them using the online editor. not that it really matters since they are commented out.